MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -13,6 +13,8 @@ final class MapViewModel: ObservableObject {
     @Published var selectedFarmer: Farmer?
     @Published var allFarmers: [Farmer] = []
     @Published var mapCameraPosition: MapCameraPosition = .userLocation(fallback: .automatic)
+    @Published var isErrorSearchFarmerPresented = false
+    @Published var titleError = ""
```

Comment:
errorTitle

Reply:
Oh my god. It’s my French side

Reply:
Same above: isSearchFarmerErrorPresented